src/plugins/dashboard/server/plugin.ts

```diff
@@ -50,13 +53,13 @@ export class DashboardPlugin
   public setup(core: CoreSetup<StartDeps>, plugins: SetupDeps) {
     this.logger.debug('dashboard: Setup');
 
-    core.savedObjects.registerType(
-      createDashboardSavedObjectType({
+    plugins.userContent.registerContent({
```

Comment:
Swap savedObjects.registerType with userContent.registerContent and we start gathering metadata information for this saved object type.